### ISSUE TITLE
Converting Data Structure

### ISSUE THREAD

**Issue by Hadkins2** [BrianGarland/vscode-rpgfree#28]
This is a similar to Issue #27.

![image](https://user-images.githubusercontent.com/95584606/148300955-4c82096b-960c-44bd-b1b2-82c265221a92.png)

In this conversion all fields defaulting to A are not converted as character.  Also packed fields are converting as the actual stored length, not packed length.  @#C004 should be packed(9:0) and @#C006 should be packed(11:2).  There was no End-DS added.  These are easy to fix and way easier than converting the DS by hand, but I wanted to let you know how the tool is working.

Here is the original data structure:

     D @@COMP          DS                  INZ
     D  @#C001              0001   0001
     D  @#C002              0002   0003B 0
     D  @#C003              0004   0008
     D  @#C004              0009   0013P 0
     D  @#C005              0014   0017P 0
     D  @#C006              0018   0023P 2
     D  @#C007              0024   0029P 2
     D  @#C008              0030   0035P 2
     D  @#C009              0036   0063
     D  @#C010              0064   0091

Thanks,

Howard Adkins

**Comment by BrianGarland** [comment]
Thank you for reporting this.  I know the LEN() is from a recent change.  The issue with packed may have always been there.

**Comment by Hadkins2** [comment]
Also found out it was doing the same thing with BINDEC as it did with PACKED. @#C002 should have been BINDEC(4).

**Comment by BrianGarland** [comment]
0.0.20 will have the len vs char and packed fix.  Are you sure about bindec?  From what I see when looking up the conversion 2 would be correct for that.  

As far as not having the END-DS, I think you might not have included the blank line that follows when doing the conversion.   There is an issue in that it only converts full lines of text.

**Comment by Hadkins2** [comment]
The END-DS issue was resolved by selecting a blank line after the original DS.  I never thought to do that, I had just selected the lines with code.

When I compiled the original program (in RPG III) variable @#C002 compiles as B(4,0).
![image](https://user-images.githubusercontent.com/95584606/148309746-29f2343b-178d-4fbc-a86b-9bd53aac1f5f.png)

When I compiled it with the DCL-DS conversion as BINDEC(2) it is only B(2,0).
![image](https://user-images.githubusercontent.com/95584606/148309864-098d5a7c-c1f9-4b83-830b-b6eccb4c7b1a.png)
